packages/api/src/external/carequality/command/cq-directory/parse-cq-directory-entry.ts

```diff
@@ -1,224 +0,0 @@
-import { ORG_POSITION, TRANSACTION_URL } from "@metriport/carequality-sdk/common/util";
-import { Address } from "@metriport/carequality-sdk/models/address";
-import { Contained } from "@metriport/carequality-sdk/models/contained";
-import { ManagingOrganization, Organization } from "@metriport/carequality-sdk/models/organization";
-import { Coordinates } from "@metriport/core/external/aws/location";
-import { out } from "@metriport/core/util/log";
-import { capture } from "@metriport/core/util/notifications";
-import { errorToString, isValidUrl, normalizeOid, normalizeZipCodeNew } from "@metriport/shared";
-import { CQDirectoryEntryData } from "../../cq-directory";
-import { CQOrgUrls } from "../../shared";
-
-const { log } = out(`parseCQDirectoryEntries`);
-
-export type LenientAddress = {
-  addressLine?: string | undefined;
-  city?: string | undefined;
-  state?: string | undefined;
-  zip?: string | undefined;
-};
-
-const EARTH_RADIUS = 6378168;
-const XCPD_STRING = "ITI-55";
-const XCA_DQ_STRING = "ITI-38";
-const XCA_DR_STRING = "ITI-39";
-const XDR_STRING = "ITI-41";
-type ChannelUrl = typeof XCPD_STRING | typeof XCA_DQ_STRING | typeof XCA_DR_STRING;
-
-export function parseCQDirectoryEntries(orgsInput: Organization[]): CQDirectoryEntryData[] {
```

Comment:
Now we parse FHIR Organizations, so this has been refactored to [parseCQOrganization](https://github.com/metriport/metriport/pull/3131/files#diff-59870e46f472301c4f3bf527b5a922ed8f32fb3bd3eeff160e5e679be49c76ddR25)